decidim-comments/spec/events/decidim/comments/user_group_mentioned_event_spec.rb

```diff
@@ -8,7 +8,7 @@
   let(:event_name) { "decidim.events.comments.user_group_mentioned" }
   let(:ca_comment_content) { "<div><p>Un commentaire pour #{author_link}</p></div>" }
   let(:en_comment_content) { "<div><p>Comment mentioning some user group, #{author_link}</p></div>" }
-  let(:author_link) { "<a class=\"user-mention\" href=\"http://#{organization.host}:#{Capybara.server_port}/profiles/#{group.nickname}\">@#{group.nickname}</a>" }
+  let(:author_link) { "<a href=\"http://#{organization.host}:#{Capybara.server_port}/profiles/#{group.nickname}\" class=\"user-mention\" target=\"_blank\" rel=\"nofollow noopener noreferrer ugc\">@#{group.nickname}</a>" }
```

Comment:
Well, not necessarily in the scope but since we are touching the `LinkRenderer` in this PR (which is necessary), this is also affected by that change.